src/doc/src/reference/workspaces.md

```diff
@@ -33,6 +33,7 @@ In the `Cargo.toml`, the `[workspace]` table supports the following sections:
 ## The `[workspace]` section
 
 To create a workspace, you add the `[workspace]` table to a `Cargo.toml`:
+
```

Comment:
This is not necessary. If you'd like to format the content, please do it in a separate commit.